tasks/skylab/StarAlign.wdl

```diff
@@ -65,3 +65,69 @@ task StarAlignBamSingleEnd {
   }
 
 }
+
+task StarAlignFastqPairedEnd {
+  input {
+    File fastq1
+    File fastq2
+    File tar_star_reference
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-star:v2.7.8a"
+    Int machine_mem_mb = ceil((size(tar_star_reference, "Gi")) + 6) * 1100
+    Int cpu = 16
+    # multiply input size by 2.2 to account for output bam file + 20% overhead, add size of reference.
+    Int disk = ceil((size(tar_star_reference, "Gi") * 2.5) + (size(fastq1, "Gi") * 5.0))
+    # by default request non preemptible machine to make sure the slow star alignment step completes
+    Int preemptible = 0
```

Comment:
TODO: Consider if we can use preemptibles here to reduce cost.

Reply:
Yes let's not merge this without adding at least 1 preemptible.

Reply:
added 3.